Tests/EncoderDecoderTests/ConverterTests.swift

```diff
@@ -20,6 +20,17 @@ import AVFoundation
 
 class ConverterTests: XCTestCase {
 
+    func testInvalidSampleRate() {
+        let src = Bundle.module.url(forResource: "InvalidSampleRate", withExtension: "ogg")!
+        let dest = URL(fileURLWithPath: NSTemporaryDirectory() + "VoiceRecordingWebOut.m4a")
+        do {
+            try OGGConverter.convertOpusOGGToM4aFile(src: src, dest: dest)
+            XCTAssert(try dest.checkResourceIsReachable(), "destination m4a file does not exist")
+        } catch {
+            XCTAssert(false, "Failed to convert from ogg to m4a with error \(error)")
```

Comment:
Not important, but could this one be an `XCTFail`, I find those a bit quicker to process in my head as "this is the bad path".